docs/getting-started.md

```diff
@@ -1,372 +1,273 @@
 # Getting Started with Firecracker
 
-## Contents
-
-- [Prerequisites](#prerequisites)
-- [Getting the Firecracker Binary](#getting-the-firecracker-binary)
-- [Running Firecracker](#running-firecracker)
-- [Building From Source](#building-from-source)
-- [Running the Integration Test Suite](#running-the-integration-test-suite)
-- [Appendix A: Setting Up KVM Access](#appendix-a-setting-up-kvm-access)
-- [Appendix B: Setting Up Docker](#appendix-b-setting-up-docker)
-
 ## Prerequisites
 
-If you need an opinionated way of running Firecracker, launch an `i3.metal`
-instance using Ubuntu 18.04 on EC2. Firecracker uses
-[KVM](https://www.linux-kvm.org) and needs read/write access that can be
-granted as shown below:
-
-```console
-sudo setfacl -m u:${USER}:rw /dev/kvm
-```
-
-The generic requirements are explained below:
-
-- **Linux**
-
-  Firecracker currently supports physical Linux **x86_64** and **aarch64**
-  hosts. The currently supported kernel versions can be found [here](kernel-policy.md).
+You can check if your system meets the requirements by running
+`firecracker/tools/devtool checkenv`.
 
-- **KVM**
+An opinionated way to run Firecracker is to launch an
+[EC2](https://aws.amazon.com/ec2/) `i3.metal` instance with Ubuntu 18.04.
```

Comment:
nit: maybe we should recommend ubuntu 22, since that's what our build container uses? (also why is this "opinionated"? :D)

Reply:
Maybe briefly mention that we use `.metal` because we don't support nested virtualization?

Reply:
Added a following comment:

> Firecracker does not support nested virtualization. This is why `.metal`
> instances are used.

Reply:
To my understanding, the reason why we use `.metal` instances is that firecracker requires KVM access and non-metal instances do not support nested virtualization.

Reply:
Updated comment to:

> Nested virtualization is not supported on non-metal EC2 instances. This is why
> `.metal` instances are used.